src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../App';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user, id } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={`TodoInfo ${completed && 'TodoInfo--completed'}`}
```

Comment:
Consider will be better if you use the classnames library for add classes with condition